scripts/image-builder/cleanup.sh

```diff
@@ -79,19 +79,10 @@ restart_osbuilder_services() {
         return
     fi
 
-    if ! systemctl is-active -q osbuild-composer.socket &>/dev/null ; then
+    if ! systemctl is-active -q osbuild-composer.service &>/dev/null ; then
         title "Starting osbuild services"
-        sudo systemctl start osbuild-composer.socket
-        for try in $(seq 3); do
-            sleep 1 # Give composer some time to be ready for workers
-            if sudo systemctl start osbuild-worker@1.service; then
-                break
-            else
-                if [ "${try}" -eq 2 ]; then
-                    return 1
-                fi
-            fi
-        done
+        sudo systemctl start osbuild-worker@1.service
+        sudo systemctl start osbuild-composer.service
```

Comment:
Should the start order be inverted here as we're first stopping workers now?

Reply:
Actually I found that we don't even need to start composer. Thanks to the socket, starting workers will cause composer to start but left it just in case